app-backend/database/src/main/scala/com/azavea/rf/database/tables/Buckets.scala

```diff
@@ -78,30 +78,32 @@ object Buckets extends TableQuery(tag => new Buckets(tag)) with LazyLogging {
     *
     * @param bucketId UUID bucket to request scenes for
     */
-  def listBucketScenes(bucketId: UUID, pageRequest: PageRequest, combinedParams: CombinedSceneQueryParams)
```

Comment:
I don't think this should be renamed since every other `list<X>` method has a `combinedParams`